server/src/Api/Handlers.hs

```diff
@@ -42,18 +45,27 @@ import Types (
   ApplyArgsRequest (ApplyArgsRequest, args, script),
   Blake2bHash (Blake2bHash),
   BytesToHash (BytesToHash),
+  CardanoError (
+    AcquireFailure,
+    EraMismatchError,
+    ScriptExecutionError,
+    TxValidityIntervalError
+  ),
   Cbor (Cbor),
   CborDecodeError (InvalidCbor, InvalidHex, OtherDecodeError),
-  CtlServerError (CborDecode),
+  CtlServerError (CardanoError, CborDecode),
   Env (protocolParams),
+  ExecutionUnitsMap (..),
```

Comment:
We should use explicit constructor/field imports. I see that we weren't actually following that convention 100% in this module, would you mind updating the other ones as well?